src/modules/transaction/store/actions.js

```diff
@@ -115,28 +116,19 @@ export const transactionBroadcasted =
       // https://github.com/LiskHQ/lisk-desktop/issues/4698 should handle this logic
     }
 
-    if (dryRunResult.data?.result === -1) {
-      dispatch({
-        type: actionTypes.broadcastedTransactionError,
-        data: {
-          error: dryRunResult.data?.errorMessage,
-          transaction,
-        },
-      });
-    }
+    const transactionErrorMessage =
+      error?.message ||
+      (dryRunResult?.data?.result === 0
```

Comment:
meaning the enum thing can now be consumed here to as 
```suggestion
      (dryRunResult?.data?.result === dryRunResultTypes.ERROR_0
```

Reply:
I believe this would make it easy for any one reading the code to know what 1 or 0 means at first glance without trying to figure out the Boolean logic.